packages/trip-details/src/fare-table.tsx

```diff
@@ -152,16 +162,14 @@ const FareTypeTable = ({
                     )
                   }
                 >
-                  {"transferAmount" in fare &&
-                    fare?.transferAmount?.amount > 0 && (
-                      <>
-                        <TransferIcon size={16} />{" "}
-                      </>
-                    )}
-                  {renderFare(
-                    fare?.price?.currency?.code,
-                    fare?.price?.amount || 0
+                  {transferAmount?.amount > 0 && (
+                    <>
+                      <TransferIcon size={16} />{" "}
+                    </>
                   )}
+                  {price
+                    ? renderFare(price?.currency?.code, price?.amount)
+                    : "-"}
```

Comment:
So "-" means that it is not possible for those riders to use the indicated payment methods? Should we announce it that way for screen readers?